src/app/meta-pixel.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { NavigationEnd, Router } from '@angular/router';
+
+import { environment } from '../environments/environment';
+
+declare const fbq: (action: string, eventOrId: string, params?: object) => void;
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MetaPixelService {
+  constructor(private router: Router) {}
+
+  init() {
+    if (!environment.metaPixelId) {
```

Comment:
would this be easier to test etc if we made the metaPixelId a constructor param instead of using it from the environment directly? Not sure.

Reply:
I think that given the constructor has only an injected param at the moment, this is maybe not worth the work to change how it's instantiated, given the functionality is not critical for the journey and the class not currently unit tested